src/compiler/program.ts

```diff
@@ -896,6 +938,26 @@ namespace ts {
                 return oldProgram.structureIsReused = StructureIsReused.Not;
             }
 
+            // Check if any referenced project tsconfig files are different
+            if (createProgramOptions.projectReferences) {
+                for (let i = 0; i < createProgramOptions.projectReferences.length; i++) {
+                    const oldRef = resolvedProjectReferences[i];

```

Comment:
How is resolvedProjectReferences[i] is an oldRef?

Reply:
Good catch - fixed